### DISCUSSION TITLE
Validating Feed error

### DISCUSSION THREAD

**Discussion by skyesweb** [rrirower/Channel-Feed-Manager#85] [General]
This is the error your software displays when testing. I started to use your software to create the feed but noticed that perhaps its the 'special attributes' that I add because they aren't available on your software. So the link to my CDN where my feed lives is in the screenshot. The special attributes that I add (songName, socialTag etc) to the json file all have corresponding elements in the screengraph and works when I sideload locally but perhaps its not happy during validation??

Maybe its Cloudinary?

![feedmanagererror](https://github.com/rrirower/Channel-Feed-Manager/assets/13589906/22bd6c00-0f27-4736-b4fe-9d9e781b0f1e)

**Comment by rrirower** [comment]
Have you checked the log file for additional info?  It's in the Documents folder.

Additionally, it appears you may have an invalid file name. The software expects to find and save a file on your hard drive, not a URL for your cloud storage.

**Reply by rrirower** [reply]
Have you had a chance to follow up on this?

**Comment by skyesweb** [comment]
No because it's looking like Roku's validator simply doesn't work after
doing more research. I got a hold of the guy at ITVC and he says he "emails
the feeds to his contact at Roku. You wouldn't happen to know how to get
ahold of someone over there?  I've already sent multiple emails to
***@***.*** and crickets. It's a black hole over there says my
DaCast rep. Arggg

Thnx

**Reply by rrirower** [reply]
Sorry, I don't have any contacts at Roku.

**Comment by rrirower** [comment]
No action from the user.
